src/migration/PoC_dfusion.js

```diff
@@ -1,7 +1,9 @@
-const { isDevelopmentNetwork, getDependency } = require("./utilities.js")
+const { isDevelopmentNetwork, getArtifactFromNpmImport, getArtifactFromBuildFolderOrImport } = require("./utilities.js")
```

Comment:
Why do we need both methods? Can't we just have on getArtifact method that behaves like getArtifactFromBuildFolderOrImport ?

Reply:
If you would use `getArtifactFromBuildFolderOrImport` for the libraries for rinkeby, then it would not find the address of the migrated contracts, as it would just use the local artifacts from the build folder and the address are only in the artifacts of the npm package.

One could combine both functions `getArtifactFromBuildFolderOrImport` and `getArtifactFromNpmImport` into a new one that would check in the try statement, whether the contracts are also deployed in the network. It would only use the artifacts in the build folder if they exist with a deployed address. I did it in the other PR, but it was awful to read or explain.